config/v1/tests/networks.config.openshift.io/NetworkDiagnosticsConfig.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Network"
+crdName: networks.config.openshift.io
+featureGate: NetworkDiagnosticsConfig
+tests:
+  onCreate:
```

Comment:
Add a test to be sure that an empty object is creatable.

Reply:
Done.